scripts/install_narrative_docker.sh

```diff
@@ -27,12 +27,15 @@ source activate base
 console "Installing biokbase requirements from src/requirements.txt"
 pip install -r $NARRATIVE_ROOT_DIR/src/requirements.txt
 
+# overwrite existing pyyaml installation (from distutils, so pip cannot uninstall it)
+pip install --ignore-installed -r $NARRATIVE_ROOT_DIR/src/requirements-ignore-installed.txt
```

Comment:
this was causing the first run of `pip install` to fail => run it separately and overwrite the existing pyyaml installation since pip can't uninstall it